tests/m2m_through_regress/models.py

```diff
@@ -40,7 +40,7 @@ class Car(models.Model):
     drivers = models.ManyToManyField('Driver', through='CarDriver')
 
     def __str__(self):
-        return "%s" % self.make
+        return str(self.make)
```

Comment:
`make` is a `CharField` so `str()` is unnecessary.

Reply:
In this instance it is nullable so we have to call `str()` as the return type of `__str__()` must be `str`.